### DISCUSSION TITLE
web.snapchat.com: ads

### DISCUSSION THREAD

**Discussion by Win-ss** [uBlockOrigin/uAssets#25151] [General]
### Prerequisites

- [X] This is **NOT** a [_YouTube_](https://www.github.com/uBlockOrigin/uAssets/issues/20586), [_Facebook_](https://www.github.com/uBlockOrigin/uAssets/issues/3367), [_Twitch_](https://www.github.com/uBlockOrigin/uAssets/issues/5184) or [_a shortener/hosting site_](https://github.com/uBlockOrigin/uAssets/discussions/17361) report. _These sites MUST be reported by clicking their **respective links**_.
- [X] I read and understand the [policy about what is a valid filter issue](https://github.com/uBlockOrigin/uAssets/blob/master/README.md#uassets).
- [X] I verified that this issue is not a duplicate. (Use this [button](https://user-images.githubusercontent.com/585534/146582579-c32707a0-36a1-4cc5-ad50-83172c9f67a8.png) to find out.)
- [X] I did not remove any of the [default filter lists](https://github.com/uBlockOrigin/uAssets/assets/66517106/6e8d024a-5869-47cf-b142-b2357eb6c7d0), or I have verified that the issue was not caused by removing any of the default lists.
- [X] I did not enable additional filter lists, or I have verified that the issue still occurs without enabling additional filter lists.
- [X] I do not have custom filters/rules, or I have verified that the issue still occurs without custom filters/rules.
- [X] I am not using uBlock Origin along with other content blockers.
- [X] I have verified that the web browser's built-in content blocker/tracking protection, [network wide/DNS blocking](https://en.wikipedia.org/wiki/DNS_blocking), or my VPN is not causing the issue.
- [X] I have verified that other extensions are not causing the issue.
- [X] If this is about a breakage or detection, I have verified that it is caused by uBlock Origin and isn't a site issue.
- [ ] I did not answer truthfully to **ALL** the above checkboxes.

### URL address of the web page

`https://web.snapchat.com/86e377d3-b66e-33dc-ac69-685a508237be`

### Category

ads

### Description

Ads appear in stories

### Other extensions used

All are unrelated,
voice master
picture in picture
Silk
indie wiki buddy
return youtube dislike

### Screenshot(s)

<details>
<summary>Screenshot(s)</summary>

![chrome_SWW3oS2VYL](https://github.com/user-attachments/assets/9effb88c-b4d2-4be5-a8b3-c0024d3b27c0)
![chrome_qBhXDKu1Hz](https://github.com/user-attachments/assets/7ef8a466-e618-4bbf-ad93-5821e66de2b6)

</details>

### Configuration

<details>

```yaml
uBlock Origin: 1.59.0
Chromium: 127
filterset (summary):
 network: 153367
 cosmetic: 48043
 scriptlet: 22150
 html: 0
listset (total-discarded, last-updated):
 default:
  user-filters: 19-3, never
  ublock-filters: 39663-122, now
  ublock-badware: 10760-6, now
  ublock-privacy: 1072-2, now
  ublock-unbreak: 2467-1, now
  ublock-quick-fixes: 214-4, now
  easylist: 85624-567, now
  easyprivacy: 53055-791, now
  urlhaus-1: 28790-0, now
  plowe-0: 3545-0, now
filterset (user): [array of 19 redacted]
trustedset:
 added: [array of 13 redacted]
userSettings: [none]
hiddenSettings: [none]
supportStats:
 allReadyAfter: 352 ms (selfie)
 maxAssetCacheWait: 186 ms
 cacheBackend: indexedDB
popupPanel:
 blocked: 216
 network:
  snapchat.com: 213
  sc-prod.net: 3
 extended:
  ##.wHvEy
```
</details>

**Comment by mapx-** [comment]
@Win-ss we need an account in order to investigate your issue (and asking to install app / phone number etc)

If you want us to investigate the issue, are you willing to give us the credentials of your account? (I'll provide an email address in case you confirm it)

**Comment by Win-ss** [comment]
@mapx- wont be able to sadly, + you will need to be on the mobile app when someone tries to login to verify it(fyi, future reference if there is someone willing to share)

**Comment by D4niloMR** [comment]
HAR file could also help https://toolbox.googleapps.com/apps/har_analyzer/

**Comment by mapx-** [comment]
I have now a snapchat account, but your link above redirects to `web.snapchat.com`

**Comment by Win-ss** [comment]
yes, the web version of snapchat
like web.whatsapp.com

**Comment by stephenhawk8054** [comment]
I think mapx means the URL at 1st post `https://web.snapchat.com/86e377d3-b66e-33dc-ac69-685a508237be` just redirects to `web.snapchat.com`

**Comment by Win-ss** [comment]
@stephenhawk8054 oh yes, that was the user id of someone i was chatting with and not related to the ad.

**Comment by mapx-** [comment]
we need a way to reproduce those ads, can you provide a workable example ?

**Comment by Win-ss** [comment]
@mapx- you need to view stories sent by other people

**Comment by mapx-** [comment]
converting in discussion, I have no clue how to reproduce the issue, maybe someone will be able to do it
